tenable/io/v3/vm/remediation_scans/schema.py

```diff
@@ -0,0 +1,42 @@
+'''
+Schema for remediation scans endpoint
+'''
+from marshmallow import INCLUDE, Schema, fields, post_dump, validates
+
+
+class RemScansDocumentCreateSchema(Schema):
+
+    class Meta:
+        unknown = INCLUDE
+
+    name = fields.Str()
+    template = fields.Str()
+    scanner = fields.Str()
+    targets = fields.List(fields.Str())
+    credentials = fields.Dict()
+    compliance = fields.Dict()
+    enabled_plugins = fields.List(fields.Int())
```

Comment:
Coulkd you please verify once if the policy_id here is int or UUID ?

Reply:
![image](https://user-images.githubusercontent.com/93529282/149269470-55a4a70f-abd9-4c23-afd4-322e30a7a01c.png)
they have not updated the id from int to uuid. so for now i have kept it in integer form.